Example/Example.Android/Resources/drawable/ic_phonelink_lock.svg

```diff
@@ -0,0 +1,4 @@
+<svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
SVGs are now both in Example and Example.Android. I think it that they should be only in Example.